docs/docs/user/run-node/03-txs.md

```diff
@@ -179,6 +179,42 @@ func sendTx() error {
 
 At this point, `TxBuilder`'s underlying transaction is ready to be signed.
 
+### Generating an Unordered Transaction
+
+Starting with Cosmos SDK v0.53.0, users may send unordered transactions to chains that have the feature enabled.
+
+Using the example above, we can set the required fields to mark a transaction as unordered. 
+By default, unordered transactions charge an extra 2240 units of gas to offset the additional storage overhead that supports their functionality.
+The extra units of gas are customizable and therefore vary by chain, so be sure to check the chain's ante handler for the gas value set, if any.
+
+```go
+import (
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+func sendTx() error {
+    // --snip--
+	txBuilder.SetUnordered(true)
+	txBuilder.SetTimeoutTimestamp(time.Now())
+}
```

Comment:
I don't think i understand this snippet.  Plus wont this tx be expired by the time it hits the node because the timeout is `time.Now()`?

Reply:
fixed the timeout also, lol brainfart